### PR TITLE
Update portuguese translation

### PR DESCRIPTION
**Pull request by maiconcarraro**
What does this PR achieve? **fix**

### Technical
Multiple changes for portuguese translation and added support to "Donate" translation for site/alert.html.
Note: I restored "Open Library" and "Internet Archive" names that were translated, it creates confusion and checking other translations files they preserved the originals.

### Testing
Run steps from `i18n/readme.md` using `?lang=pt` query param.

### Screenshot
[Before](https://user-images.githubusercontent.com/13419087/214093247-5f7d3e97-87bc-4838-9cc6-d964ab50b804.png) and [After](https://user-images.githubusercontent.com/13419087/214092786-138e0cef-d5d7-4530-ba3e-efec3008298a.png)

### CODE REVIEW

File: openlibrary/i18n/pt/messages.po [hunk 1]
```diff
@@ -650,7 +650,7 @@ msgstr "Emprestar"
 #: widget.html:45
 #, python-format
 msgid "Join waitlist for &quot;%(title)s&quot;"
-msgstr "Entrar na lista de espera para «%(title)s»"
+msgstr "Entrar na lista de espera para &quot;%(title)s&quot;"
```
Comment (cdrini) [inline openlibrary/i18n/pt/messages.po:653]: We can't use html entities like `&quot;` everywhere, so using the direct symbol is better.

File: openlibrary/i18n/pt/messages.po [hunk 2]
```diff
@@ -1980,7 +1985,7 @@ msgid ""
 "Like, \"Agatha Christie\" or \"Jean-Paul Sartre.\" We'll also do a live "
 "check to see if we already know the author."
 msgstr ""
-"Como «Agatha Christie» ou «Jean-Paul Sartre». Também vamos verificar ao vivo "
+"Como \"Agatha Christie\" ou \"Jean-Paul Sartre\". Também vamos verificar ao vivo "
```
Comment (cdrini) [inline openlibrary/i18n/pt/messages.po:1988]: What is the correct quoting to use for Portuguese?

Comment (ninalencastro) [reply openlibrary/i18n/pt/messages.po:1988]: It would be " as it is in English, if I understood the question correctly.

**Comment by ninalencastro** [comment]
I agree, maintaing the names in English is a good change. All in order with the changes made :)

File: openlibrary/i18n/pt/messages.po (hunk 1 above)
Comment (cdrini) [reply openlibrary/i18n/pt/messages.po:653]: Ah sorry missed it was that way in the `msgid`! So it will work!

**Review by cdrini — APPROVED** [review]
Lgtm! Thank you @maiconcarraro ! And thank you for the review, @ninalencastro ! :)

**Pull request merged**